extensions/positron-python/src/client/positron/webAppContexts.ts

```diff
@@ -15,6 +26,7 @@ export function detectWebApp(document: vscode.TextDocument): void {
 }
 
 export function getFramework(text: string): string | undefined {
+    const libraries = getSupportedLibraries();
```

Comment:
where this is placed means that users will have to restart positron to not have duplicated buttons, does this seem like an okay cadence?